src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -368,22 +368,21 @@ <h2 class="course-detail__title">{% blocktrans context "course_detail__title" %}
             {% block runs %}
                 <div class="course-detail__aside">
                     {% render_model_add course "" "" "get_admin_url_to_add_run" %}
-                    {% with runs_dict=current_page.course.course_runs_dict %}
-                        {% with open_visible_runs=runs_dict.0|add:runs_dict.1|add:runs_dict.2|visible_on_course_page:request.toolbar.edit_mode_active %}
-                            <!--
-                                As course runs are rendered through React, we use meta tags to
-                                setup course runs RFDa properties.
-                             -->
-                            {% for run in open_visible_runs %}
-                                <span rel="hasCourseInstance" typeof="CourseInstance">
-                                    <meta property="name" content="{{ run.title }}"  />
-                                    <meta property="inLanguage" content="{{ run.get_languages_display }}"  />
-                                    <meta property="courseMode" content="online"  />
-                                    <meta property="startDate" content="{{ run.start|date:'Y-m-d' }}"  />
-                                    <meta property="endDate" content="{{ run.end|date:'Y-m-d' }}"  />
-                                </span>
-                            {% endfor %}
-                        {% endwith %}
+                    {% with runs=current_page.course.course_runs|visible_on_course_page:request.toolbar.edit_mode_active %}
+                        <!--
```

Comment:
There is an issue with that as we will render something for to be scheduled runs.

Reply:
If you mean about the course runs without any start, end, enrollment start or end dates, the Google Search Console Rich Test Results online validator didn't complain if that information has an empty content. From the spec I don't know what should be put on the html if the course is on the to be scheduled state. I think we should write it without any date value, and this is what the application is doing.

Reply:
Yup I'm talking about this kind of course run. But currently those course run are not shown on the published course page. And with your change, the course run will not be shown but a rdfa meta tag will be rendered.

Reply:
@jbpenrath Ok, now I understand. I've updated the PR so the course runs "to be scheduled" won't be included on the RFDa.
I have included a "to be scheduled" run on the RDFa test `test_templates_course_detail_rdfa`, so we secure it better.

Nevertheless, I've found another problem! The course runs "to be scheduled" with a new language (and enrollment count number) change the course available languages and its enrollment count. This isn't specific to RDFa.
That is why I've included a couse run without an existing language (from other previous exiting run) and with a `0` enrollment count.
```python
        CourseRunFactory(
            title="A to be scheduled run",
            direct_course=course,
            start=None,
            end=None,
            enrollment_start=None,
            enrollment_end=None,
            languages=["en"],
            enrollment_count=0,
        )
```

If you agree I can open an issue for that problem of course runs "to be scheduled" that should be hidden, but are changing the languages and enrollment count.

Reply:
@jbpenrath I think this is resolved.